src/base/oven.lua

```diff
@@ -695,23 +695,40 @@
 -- conflicting object file names - hello1.o
 
 	function oven.uniqueSequence(f, cfg, seq, bases)
-		while true do
+		local good_sequence = true
+		repeat
```

Comment:
Not the end of the world or anything, but what was the reason for changing this?